swagger.yaml

```diff
@@ -30,6 +26,22 @@ paths:
           schema:
             $ref: '#/definitions/Info'
 
+  /departments:
+    get:
+      summary: List departments
+      description: Return org code and org description (departments) for a business center
+      parameters:
+        - in: query
```

Comment:
If its just a single business center at a time would a path parameter make more sense? For example:
`/departments/{4-character business name}`
and then /departments could be reserved for a GET endpoint returning the whole collection?